packages/commerce-sdk-react/jest.config.js

```diff
@@ -9,6 +9,7 @@ const base = require('internal-lib-build/configs/jest/jest.config')
 
 module.exports = {
     ...base,
+    setupFilesAfterEnv: ['./setup-jest.js'],
```

Comment:
Oh that's how you do it.. TIL about `setupFiles` vs `setupFilesAfterEnv`: https://jestjs.io/docs/configuration#setupfilesafterenv-array

> In other words, setupFilesAfterEnv modules are meant for code which is repeating in each test file.